reference/twig-extensions/functions/get_media_url.rst

```diff
@@ -0,0 +1,25 @@
+``get_media_url``
+================
+
+Returns relative URL to the given media. Configuration is needed if this Twig Extension is used!
+
+**Configuration**:
+
+Following configuration means, that the default dispositionType is "attachment" for each file and only if the mimeTypes of a file match "application/pdf" or "image/jpeg" it's the "inline" dispositionType. 
+
+If the default dispositionType would be "inline" and some files should be "attachment", than the configuration of "mime_types_attachment" should be filled and "mime_types_inline" should be empty.
+
+.. code-block:: jinja
```

Comment:
`jinja` is the templating language twig is inspired from, you should use `yaml` instead